tests/integrationtest/r/executor/issues.result

```diff
@@ -899,43 +899,43 @@ TOTAL_SPLIT_REGION	SCATTER_FINISH_RATIO
 203	1
 analyze table pt;
 set @@tidb_distsql_scan_concurrency = default;
-explain analyze select * from t order by id;  # expected distsql concurrency 2
```

Comment:
Comment is not displayed in the result file?  @Defined2014

Reply:
Should use `--echo xxx` to display it. Previously, comments on a single line were not displayed.